### PR TITLE
SCJ-234: Update question wording

### PR DESCRIPTION
**Pull request by duncan-oxd**
Update to the wording on one of the form questions

### CODE REVIEW

File: app/Views/ScBooking/BookingType.cshtml [hunk 1]
```diff
@@ -111,9 +111,8 @@
 
                     <div id="different-place-of-trial" class="form-group btn-radio-group mb-5">
                         <h6 class="form-section-label">
-                            Do you have an entered court order changing the place of trial or is a different place of
-                            trial stated on the initiating documents (e.g. Notice of Civil Claim, Notice of Family
-                            Claim)?
+                            Do you have an entered court order changing the place of trial, or is this place of trial
+                            stated on the initiating documents (e.g. Notice of Civil Claim, Notice of Family Claim)?
```
Comment (molund) [inline app/Views/ScBooking/BookingType.cshtml:115]: I wonder if they realized when they were writing this sentence that the form is progressively revealed?

e.g. "...is **this** place of trial..." <= what does "this" refer to?

**Review by molund — APPROVED** [review]
The PR looks good to me, but I am a bit concerned that the might have written the sentence based on a misleading wireframe or screenshot.

File: app/Views/ScBooking/BookingType.cshtml (hunk 1 above)
Comment (duncan-oxd) [reply app/Views/ScBooking/BookingType.cshtml:115]: Updating here so I don't forget: Dom passed the ticket back to Winnie to see if we can clarify the wording a bit. That's what I expected so I've held off merging until the wording is revised 👍

**Comment by duncan-oxd** [comment]
Rebased and updated the wording for the latest version approved by Lorne. Gonna merge this now and pass it back to be tested

**Pull request merged**